include/oxli/storage.hh

```diff
@@ -421,7 +421,11 @@ public:
     // in the key (current value of size+8 is copied from the CQF example)
     // Final argument is the number of bits allocated for the value, which
     // we do not use.
-    qf_init(&cf, (1ULL << size), size+8, 0);
+    _supports_bigcount = true;
+    qf_init(&cf, (1ULL << size), size+8, 0,2,true,"",2038074761);
```

Comment:
There is no documentation for the initialization functions at mqfDevelopmenet!

Reply:
Also if I understand this correctly, "cf" is your variable name. Let's use "mf" instead to make it more representative.